pkg/logs/input/container/launcher.go

```diff
@@ -26,28 +26,28 @@ import (
 // If none of those volumes are mounted, returns a lazy docker launcher with a retrier to handle the cases
 // where docker is started after the agent.
 // dockerReadTimeout is a configurable read timeout for the docker client.
-func NewLauncher(collectAll bool, collectFromFiles bool, dockerReadTimeout time.Duration, sources *config.LogSources, services *service.Services, pipelineProvider pipeline.Provider, registry auditor.Registry) restart.Restartable {
+func NewLauncher(collectAll bool, kubernetesCollectFromFiles bool, dockerCollectFromFiles bool, dockerReadTimeout time.Duration, sources *config.LogSources, services *service.Services, pipelineProvider pipeline.Provider, registry auditor.Registry) restart.Restartable {
```

Comment:
I think we should update the method doc to reflect the new parameters. ex:
```
// collectFromFiles is enabled.
```
is no longer accurate.